css/css-color/border-color-currentcolor.html

```diff
@@ -0,0 +1,21 @@
+<style>
```

Comment:
In order for this to be run as a reftest, this needs a `<link rel=match>` as documented in https://web-platform-tests.org/writing-tests/reftests.html .

I'd suggest it might also be good to add some of the other metadata that you see in other tests in this directory (e.g., `title` element, author, link to relevant section of spec.